browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -2065,6 +2065,61 @@ L.CanvasTileLayer = L.Layer.extend({
 		}
 	},
 
+    _highlightColAndRow: function (textMsg, part) {
```

Comment:
this function is needed only in Calc, please move to `CalcTileLayer` so we can do simplification I mentioned in `CanvasTileLayer`